apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/jdk8/pathAssertions.kt

```diff
@@ -156,6 +156,29 @@ val <T : Path> Expect<T>.parent: Expect<Path>
 fun <T : Path> Expect<T>.parent(assertionCreator: Expect<Path>.() -> Unit): Expect<T> =
     ExpectImpl.path.parent(this).addToInitial(assertionCreator)
 
+/**
+ * Converts the given [other] to a path and expects it to resolve against this [Path]. Creates an [Expect] for it,
```

Comment:
This is slightly wrong as you don't convert `other` to a path first. Also I would stick to other docs and not split it up in two sentences. I suggest:
```suggestion
 * Expects that [other] resolve against this [Path] and creates an [Expect] for the resolved [Path]
```